extensions/control-plane/store/sql/transfer-process-store-sql/src/main/java/org/eclipse/edc/connector/store/sql/transferprocess/store/schema/TransferProcessStoreStatements.java

```diff
@@ -75,6 +75,10 @@ default String getConnectorAddressColumn() {
         return "connector_address";
     }
 
+    default String getTransferTypeColumn() {
+        return "transfer_type";
```

Comment:
this needs to be added to the `TransferProcessMapping` as well, otherwise querying won't work. While doing that, I would
also suggest to add a test to query for that property in the `TransferProcessStoreTestBase`:

```java

@Test
void queryByTransferType() {
    range(0, 10).forEach(i -> getTransferProcessStore().save(createTransferProcessBuilder("test-tp-" + i)
            .transferType("type" + i)
            .build()));
    var querySpec = QuerySpec.Builder.newInstance().filter(Criterion.criterion("transferType", "=", "type4")).build();

    var result = getTransferProcessStore().findAll(querySpec);

    assertThat(result).extracting(TransferProcess::getTransferType).containsOnly("type4");
}
```

Reply:
done